target/linux/ath79/generic/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -95,12 +90,4 @@ case "$board" in
 		[ "$PHYNBR" -eq 0 ] && \
 			nvram get wl0_hwaddr > /sys${DEVPATH}/macaddress
 		;;
-	zyxel,nwa1123-ac)
-		[ "$PHYNBR" -eq 0 ] && \
-			mtd_get_mac_text mib0 0x66 > /sys${DEVPATH}/macaddress
-		;;
-	zyxel,nwa1123-ni)
-		[ "$PHYNBR" -eq 1 ] && \
-			mtd_get_mac_text mib0 0x66 > /sys${DEVPATH}/macaddress
```

Comment:
Oh this proves that PCIe is phy1 if both are ath9k NICs. Please check this PR again.